src/frameworks/TYPO3.md

```diff
@@ -0,0 +1,17 @@
+# Getting started
+
+## Prerequisites
+
+**Composer**
+
+Composer is a tool for dependency management in PHP. It allows you to
+declare the dependent libraries your project needs and it will install
+them in your project for you.
```

Comment:
Hard wrapping: Hard carriage return every 80 characters.
Soft wrapping: Hard carriage return at the end of a paragraph, so in the source it's just one long line. Let your editor handle wrapping for your editor window.